build_java.pl

```diff
@@ -169,9 +180,8 @@ sub setup_vars {
     our $jni_header_dir = "$dist_dir/private/jss/_jni";
 
     our $jarFiles = "/usr/share/java/slf4j/slf4j-api.jar:/usr/share/java/commons-codec.jar";
-    if( $ENV{DEBIAN_BUILD} ) {
-        $jarFiles = "/usr/share/java/slf4j-api.jar:/usr/share/java/commons-codec.jar";
-    }
+
+    $jarFiles = Common::get_jar_files;
```

Comment:
Make this `our $jarFiles` and remove line 182.

Reply:
Yes, I tried it and it works.